WordPress/src/main/java/org/wordpress/android/ui/reader/services/update/ReaderUpdateLogic.java

```diff
@@ -333,6 +358,9 @@ public void run() {
                         ReaderPostTable.updateFollowedStatus();
                         AppLog.i(AppLog.T.READER, "reader blogs service > followed blogs changed");
                         EventBus.getDefault().post(new ReaderEvents.FollowedBlogsChanged());
+                        // bump analytics
+                        final int totalSites = jsonObject == null ? 0 : jsonObject.optInt("total_subscriptions", 0);
+                        ReaderTracker.trackFollowedSitesCount(totalSites);
```

Comment:
⚠️ I would prefer not to add Analytics tracking logic (which I would consider business logic) inside this data layer object. I believe you can accomplish the analytics tracking in some place that hooks into the `FollowedBlogsChanged` event instead of doing all this logic inside the data fetching mechanism.